From a4c3a7440088363a1bb90ee161c29bc12cc3cc84 Mon Sep 17 00:00:00 2001
From: Jason Liu <jason.hui.liu@nxp.com>
Date: Fri, 15 Dec 2017 08:40:59 +0800
Subject: [PATCH 3111/5242] MLK-17224 arm64: dts: jdi-wuxga-lvds1-panel.dts:
 disable ldb1

commit  d2982edd8dbad43c5246530934a3b666f49e5abe from
https://source.codeaurora.org/external/imx/linux-imx.git

current dts supports dual channel LVDS panel connected with ldb2, while it
also needs the cable connected with ldb1 for example need  LVDS-HDMI covert
connect with LVDS0_CH0 on the board, otherwise, the LVDS panel not work.

This is not user friendly. This patch removes the limitation by disable the
ldb1 interface by default.

Signed-off-by: Jason Liu <jason.hui.liu@nxp.com>
Acked-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 ...sl-imx8qm-lpddr4-arm2-jdi-wuxga-lvds1-panel.dts |    8 ++++++++
 .../fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dts       |    8 ++++++++
 2 files changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-jdi-wuxga-lvds1-panel.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-jdi-wuxga-lvds1-panel.dts
index 5f9357c..0ee41a6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-jdi-wuxga-lvds1-panel.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-jdi-wuxga-lvds1-panel.dts
@@ -27,6 +27,14 @@
 	};
 };
 
+&ldb1_phy {
+       status = "disabled";
+};
+
+&ldb1 {
+       status = "disabled";
+};
+
 &ldb2_phy {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dts
index 762c400b..3dfb58a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-jdi-wuxga-lvds1-panel.dts
@@ -27,6 +27,14 @@
 	};
 };
 
+&ldb1_phy {
+	status = "disabled";
+};
+
+&ldb1 {
+	status = "disabled";
+};
+
 &ldb2_phy {
 	status = "okay";
 };
-- 
1.7.9.5

